### ISSUE TITLE
Add GSCN calculator

### ISSUE THREAD

**Issue by blevic** [blevic/nrarfcn#11]

**Comment by blevic** [comment]
Use _Table 5.4.3.1-1: GSCN parameters for the global frequency raster_

Then add APIs:
```
get_gscn_by_frequency(freq: float) -> int
get_frequency_by_gscn(gscn: int) -> float
```

**Comment by blevic** [comment]
Use _Table 5.4.3.3-1: Applicable SS raster entries per operating band (FR1)_ and _Table 5.4.3.3-2: Applicable SS raster entries per operating band (FR2)_

Then add API:
`get_gscn_range(band: Union[str, int]) -> tuple`

**Comment by blevic** [comment]
Please mention the decoded acronym "Global Synchronization Channel Number" somewhere

**Comment by blevic** [comment]
Tables included by PR #19

**Comment by blevic** [comment]
Completed by PR #22
